client/src/components/account/CreateAccount.js

```diff
@@ -36,6 +36,9 @@ class CreateAccount extends Component {
         if (res.code === "auth/weak-password") {
           alert("Your password is too weak.");
         }
+        else {
+          alert("Account successfully created!");
```

Comment:
We don't have to change this now, but when an account is successfully created we should send them to the home page signed in.